### PR TITLE
Remove css

### PR DESCRIPTION
**Pull request by swaterkamp**
Part of #17

### CODE REVIEW

File: ng/src/web/components/form/formgroup.js [hunk 1]
```diff
@@ -48,38 +90,33 @@ const FormGroup = ({
...
   return (
-    <Layout
+    <FormGroupLayout
       flex
       align={['start', 'center']}
-      className={className}>
-      {title}
-      <Layout
+      className={className}
+      >
```
Comment (bjoernricks) [inline ng/src/web/components/form/formgroup.js]: Could you remove two spaces?

File: ng/src/web/components/form/formgroup.js [hunk 2]
```diff
@@ -48,38 +90,33 @@ const FormGroup = ({
...
-  }
   return (
-    <Layout
+    <FormGroupLayout
       flex
       align={['start', 'center']}
-      className={className}>
-      {title}
-      <Layout
+      className={className}
```
Comment (bjoernricks) [inline ng/src/web/components/form/formgroup.js]: Is passing the className to FormGroup still necessary?

File: ng/src/web/components/form/formgroup.js [hunk 3]
```diff
@@ -48,38 +90,33 @@ const FormGroup = ({
...
+          {title}
+        </Title>
+      }
+      <FormGroupContent
         {...other}
         flex={flex}
-        className={css}>
+        offset={offset}
+        size={size}
+        >
```
Comment (bjoernricks) [inline ng/src/web/components/form/formgroup.js]: Here are also two spaces to much for indentation.

File: ng/src/web/components/form/formgroup.js [hunk 4]
```diff
@@ -22,14 +23,55 @@
...
+  paddingLeft: '10px',
+  paddingRight: '10px',
+} : null,
+  ({offset}) => is_defined(offset) ? {
+    marginLeft: COLUMNS[parse_int(offset) - 1],
+  } : null,
+);
 
 const FormGroup = ({
     children,
```
Comment (bjoernricks) [inline ng/src/web/components/form/formgroup.js]: The indentation of the props of FormGroup could be fixed too.

File: ng/src/web/components/form/textarea.js [hunk 5]
```diff
@@ -22,20 +23,41 @@
...
+  padding: '1px 8px',
+}, ({disabled}) =>
+    disabled ? {cursor: 'not-allowed'} : null,
+    ({disabled, readonly}) => readonly || disabled ? {
+      backgroundColor: Theme.dialogGray,
+      opacity: 1,
+    } : null,
+  );
+
 const TextAreaComponent = ({className, value = '', ...props}) => {
```
Comment (bjoernricks) [inline ng/src/web/components/form/textarea.js]: IMHO the TextAreadComponent is unnecessary now. You could remove it and use StyledTextAread instead.

File: ng/src/web/pages/tasks/dialog.js [hunk 6]
```diff
@@ -362,8 +362,8 @@ const TaskDialog = ({
             <Layout
               flex="column"
               offset="2"
-              className={classes('offset-container',
-                state.in_assets === YES_VALUE ? '' : 'disabled')}>
+              className={'offset-container ' +
```
Comment (bjoernricks) [inline ng/src/web/pages/tasks/dialog.js]: I don't think this is better then before. Maybe we should introduce a new OffsetLayout class? The offset-container css class should be removed in future too.

File: ng/src/web/components/icon/greenboneicon.js [hunk 7]
```diff
@@ -24,14 +24,13 @@
...
 import _ from 'gmp/locale.js';
-import {classes} from 'gmp/utils';
 
 import PropTypes from '../../utils/proptypes.js';
 
 import Icon from './icon.js';
 
 const GreenboneIcon = ({className, size = 'default', ...props}) => {
-  className = classes(className, 'greenbone-icon');
+  className = className + ' greenbone-icon';
```
Comment (bjoernricks) [inline ng/src/web/components/icon/greenboneicon.js]: Just remove className completely. We shouldn't add class names anymore. Only for debugging purposes in glamorous components.

Comment (bjoernricks) [reply ng/src/web/components/icon/greenboneicon.js]: It doesn't make sense to replace classses helper function by string concatenation. classes is still better and easier to use then using plain strings.

File: ng/src/web/components/form/submitbutton.js [hunk 8]
```diff
@@ -23,14 +23,12 @@
...
 
-import {classes} from 'gmp/utils';
-
 import PropTypes from '../../utils/proptypes.js';
 
 import Button from './button.js';
 
 const SubmitButton = ({className, ...other}) => {
-  className = classes('button-block', className);
+  className = className + ' button-block';
```
Comment (bjoernricks) [inline ng/src/web/components/form/submitbutton.js]: Same here. Replacing classes doesn't make sense.

File: ng/src/web/components/form/selectelements.js [hunk 9]
```diff
@@ -46,6 +46,11 @@ export const Box = glamorous.div({
 } : null);
 
 export const Input = glamorous.input({
+  /* use font and line settings from parents not from browser default */
+  fontFamily: 'inherit',
+  fontSize: 'inherit',
+  lineHeight: 'inherit',
+
```
Comment (bjoernricks) [inline ng/src/web/components/form/selectelements.js]: Just curious why is this necessary?

Comment (swaterkamp) [reply ng/src/web/components/form/selectelements.js]: The old .css had those three settings for all ```<input>```, ```<textarea>``` and ```<select>``` elements. To not lose these, I added them here.

Comment (bjoernricks) [reply ng/src/web/components/form/selectelements.js]: Yeah but the original css was derived from bootstrap and relied on other css formattings. I am really not sure if that is necessary at all anymore. Even I am unsure about the original purpose.

Comment (swaterkamp) [reply ng/src/web/components/form/selectelements.js]: Leaving it out didn't make an obvious difference for the formgroups whose function I checked during programming. But then again, I didn't check all dialogs and use cases of the formgroups especially not with changed fonts etc. I can easily just delete it. Those settings are copied to some other files of this PR as well. Should I also delete those?

File: ng/src/web/components/form/formgroup.js (hunk 2 above)
Comment (swaterkamp) [reply ng/src/web/components/form/formgroup.js]: It has been during some changes I made before, but in the end, it's just a leftover. I removed it now.
